src/Client/Client.csproj

```diff
@@ -0,0 +1,21 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+        <TargetFrameworks>netstandard2.0</TargetFrameworks>
+        <PlatformTarget>AnyCPU</PlatformTarget>
+        <AssemblyName>OmniSharp.Extensions.LanguageClient</AssemblyName>
+        <RootNamespace>OmniSharp.Extensions.LanguageServerProtocol.Client</RootNamespace>
+    </PropertyGroup>
+
+    <ItemGroup>
+        <PackageReference Include="Newtonsoft.Json" Version="10.0.3" />
+        <PackageReference Include="Serilog" Version="2.5.0" />
```

Comment:
I'm hesitant to depend on `Serilog` by itself here, as I would generally prefer to not prescribe logging frameworks for consumers.

However Serilog is a great logger, so I'm okay with it for now, and perhaps refactor it away (for MS.Extensions) in a future (breaking) change.

Reply:
Thanks for the reminder - I'm fine with changing over to MEL for logging, I just forgot about it :)

I'll modify them to take an `ILogger` / `ILoggerFactory` instead (some components need to pass a different `ILogger<TComponent>` to child components).